hm_pyhelper/hardware_definitions.py

```diff
@@ -120,6 +130,8 @@ def is_raspberry_pi() -> bool:
     # Nebra Pi 0 Light Hotspot USB Ethernet
     'NEBHNT-LGT-ZX': {
         'FRIENDLY': 'Nebra Pi 0 Light Hotspot XE',
+        'CPU_ARCH': 'armv6hf',
```

Comment:
Do these two pi zero ones need to match the architecture shown on Balena?

Reason I ask is that on BalenaOS page it shows the arch for pi zero as just `rpi` - but can't remember what exactly this is needed for

https://www.balena.io/os/

Reply:
CPU_ARCH is for our reference right now. Balena doesn't ask for arch, it only asks for device type while fleet creation. Might be useful later to do multiarch containers or picking the right binary for gateway-mfr-rs or gateway-rs etc.